planner: update docs for auto analyze (#11439)
pingcap/docs: statistics.md

When the ratio of the number of modified rows to the total number of rows of `tbl` in a table is greater than `tidb_auto_analyze_ratio`, and the current time is between `tidb_auto_analyze_start_time` and `tidb_auto_analyze_end_time`, TiDB executes the `ANALYZE TABLE tbl` statement in the background to automatically update the statistics on this table.

> To avoid the situation that modifying a small amount of data on a small table will have automatic update frequently triggered, when a table has less than 1000 rows, TiDB does not trigger automatic update for this table. You can use `SHOW STATS_META` to see the number of rows of a table.

Reviewer: ```suggestion
To avoid the situation that modifying a small amount of data on a small table frequently triggers the automatic update, when a table has less than 1000 rows, such data modifying does not trigger the automatic update in TiDB. You can use the `SHOW STATS_META` statement to view the number of rows in a table.
```
